### ISSUE TITLE
No hint path defined for [flash]

### ISSUE THREAD

**Issue by johnshepherd** [Pseudoagentur/soa-sentinel#118]
Hi,

Installed this package (development branch) and got this error after logging in to the admin panel. To resolve it I had to add `'Laracasts\Flash\FlashServiceProvider'` to the providers array in `config/app.php`. Don't have time to investigate it now so I'm not sure if something just went wrong with my install or if the reference is missing from the package service provider but thought I'd mention it.

**Comment by noxify** [comment]
dev docu is not up to date @bihlmaierh is working on this, see #65.

As workaround, please do the following:
(Copied from https://github.com/laracasts/flash)

include the service provider within config/app.php.
```
'providers' => [
    'Laracasts\Flash\FlashServiceProvider'
];
```
And, for convenience, add a facade alias to this same file at the bottom:
```
'aliases' => [
    'Flash' => 'Laracasts\Flash\Flash'
];
```

**Comment by noxify** [comment]
Will be fixed with #65
